src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -63,6 +64,15 @@
 public interface RqMultipart extends Request {
 
     /**
+     * Carriage return constant.
+     */
+    String CRLF = "\r\n";
+    /**
+     * Content type constant.
+     */
+    String CONTENT_TYPE = "Content-Type";
```

Comment:
@yegor256 we are using this two in two different class which are RqMultipart.Fake and RqMultipart.Base . If i don't define a variable qulice will not allow duplicate string. what is your suggestion ? And also they are opened just interface implementer what is the problem in here ?
thank you ?